### PR TITLE
Remove info about applying to speak. Rearrange content.

### PR DESCRIPTION
**Pull request by sublimemarch**
I removed the info about how to apply to speak and rearranged the content. It includes a line about speakers being announced soon. Figured we should do an update like this quick in case anyone is checking.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -19,25 +19,12 @@
...
-    <ul>
-      <li>Gaming and interactive storytelling media</li>
-      <li>Science and technology</li>
-      <li>Making and crafting</li>
-      <li>Art, music, and culture</li>
-      <li>Anything else you find interesting!</li>
-    </ul>
-    <p>Slide decks for talks are encouraged but not required. We will provide projectors for speakers and make slides and presenter notes available online after the event.</p>
-    <p>If you’d like to reminders about proposal deadlines and announcements for upcoming events, you should sign up for our newsletter below!</p>
+    <p>This first event is a lightning talk showcase of cool things people are learning about or involved in. Talks are 5-15 minutes along and assume no prior knowledge of the topic. We've selected our talks for this first event, and we'll be announcing them soon!</p>
```
Comment (thejessleigh) [inline index.html]: Should read `5-15 minutes long` :)

File: index.html [hunk 2]
```diff
@@ -19,25 +19,12 @@
...
+    <p>Don't want to present, and just want to attend and soak up all that awesome knowledge? That's fine, too! We encourage you to take notes, livetweet, and ask awesome questions of our speakers afterwards.</p>
```
Comment (thejessleigh) [inline index.html]: Might want to mention that a couple speakers requested not to have photos of them tweeted out? Not sure how to handle that messaging yet.

**Review by thejessleigh — APPROVED** [review]

File: index.html (hunk 2 above)
Comment (sublimemarch) [reply index.html]: Good call! I think we can just mention that at the event, early and often.

Comment (sublimemarch) [reply index.html]: I don't want to say it on the site, for fear that it'll cause readers to want to dig into who the speakers are a bit too much. I know that Codeland just mentioned it day-of.

File: index.html (hunk 1 above)
Comment (sublimemarch) [reply index.html]: Fixed in a new commit!

**Comment by thejessleigh** [comment]
LGTM 👍

**Pull request merged**